src/main/target/OMNIBUSF4/target.h

```diff
@@ -174,6 +174,10 @@
 
 #define AVOID_UART1_FOR_PWM_PPM
 
+#define SPEKTRUM_BIND
+// USART1 RX
+#define BIND_PIN                PA10
```

Comment:
Good. But how is that connected? Bypassing the Sbus inverter in some way? Do you have the schematics? Just curious ....